[2024-09-15T08:15:09.374+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: save_city_info_weather_v01.save_city_info_weather scheduled__2024-09-15T01:00:00+00:00 [queued]>
[2024-09-15T08:15:09.388+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: save_city_info_weather_v01.save_city_info_weather scheduled__2024-09-15T01:00:00+00:00 [queued]>
[2024-09-15T08:15:09.390+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-09-15T08:15:09.390+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2024-09-15T08:15:09.390+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-09-15T08:15:09.498+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): save_city_info_weather> on 2024-09-15 01:00:00+00:00
[2024-09-15T08:15:09.505+0000] {standard_task_runner.py:55} INFO - Started process 1910 to run task
[2024-09-15T08:15:09.512+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'save_city_info_weather_v01', 'save_city_info_weather', 'scheduled__2024-09-15T01:00:00+00:00', '--job-id', '354', '--raw', '--subdir', 'DAGS_FOLDER/saver_info.py', '--cfg-path', '/tmp/tmpl1n7e7yu']
[2024-09-15T08:15:09.514+0000] {standard_task_runner.py:83} INFO - Job 354: Subtask save_city_info_weather
[2024-09-15T08:15:09.686+0000] {task_command.py:389} INFO - Running <TaskInstance: save_city_info_weather_v01.save_city_info_weather scheduled__2024-09-15T01:00:00+00:00 [running]> on host 0c5fe11d0c9c
[2024-09-15T08:15:09.835+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=save_city_info_weather_v01
AIRFLOW_CTX_TASK_ID=save_city_info_weather
AIRFLOW_CTX_EXECUTION_DATE=2024-09-15T01:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-09-15T01:00:00+00:00
[2024-09-15T08:15:09.838+0000] {sql.py:253} INFO - Executing: 
            create table if not exists weathers (
                id serial PRIMARY KEY ,
                city_name VARCHAR,
                max_temp VARCHAR,
                min_temp VARCHAR,
                wind VARCHAR,
                descript VARCHAR
            )
        
[2024-09-15T08:15:09.851+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-15T08:15:10.111+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-15T08:15:10.119+0000] {sql.py:364} INFO - Running statement: 
            create table if not exists weathers (
                id serial PRIMARY KEY ,
                city_name VARCHAR,
                max_temp VARCHAR,
                min_temp VARCHAR,
                wind VARCHAR,
                descript VARCHAR
            )
        , parameters: None
[2024-09-15T08:15:10.147+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=save_city_info_weather_v01, task_id=save_city_info_weather, execution_date=20240915T010000, start_date=20240915T081509, end_date=20240915T081510
[2024-09-15T08:15:10.194+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-09-15T08:15:10.269+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
